src/components/MemberArea/EditProfile.css

```diff
@@ -48,4 +48,34 @@
 .edit-profile .select__control {
   background: none;
   border: none;
-}
\ No newline at end of file
+}
+
+.edit-profile .channel-group {
+  position: relative;
+  border: 1px solid #eee;
+  width: 100%;
+  display: flex;
```

Comment:
You can let the browser to vertical align the items instead of doing it by yourself:

1.
```suggestion
  display: flex;
  align-items: center;
```